src/ProjectTemplates/test/ProjectTemplates.Blazor.Tests/BlazorServerTemplateTest.cs

```diff
@@ -0,0 +1,93 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Is this file identical to what src/ProjectTemplates/test/BlazorServerTemplateTest.cs was before❔ GitHub may be confusing itself.
Same question about src/ProjectTemplates/test/WebApiTemplateTest.cs and src/ProjectTemplates/test/ProjectTemplates.Mvc.Tests/WebApiTemplateTest.cs

Reply:
Sorry I'm not following what the question is here. This file should be identical to the file that was in the single test project before.

Reply:
Like I said, GitHub was confuzzzzzed